package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ci-trap",
-  "version": "0.4.0",
-  "description": "Lightweight mouse movement tracker library for browsers.",
+  "version": "1.0.0",
+  "description": "Lightweight mouse and touch movement tracker library for browsers.",
```

Comment:
I'd use "mouse and touch event tracker" instead -- if we change the description.